portfolio.html

```diff
@@ -8,6 +8,284 @@
+        <img src="./images/desktop_version/frame_headline/Frame-big.png" class="desktop-frame-big" alt="Frame big icon">
+        <img src="./images/desktop_version/frame_headline/Group-37.png" class="desktop-group-37" alt="Group 37 image">
+        <img src="./images/desktop_version/frame_headline/Frame-middle.png" class="desktop-frame-middle" alt="Frame middle icon">
+        <img src="./images/desktop_version/frame_headline/Frame-right.png" class="desktop-frame-right" alt="Frame right icon">
+        <img src="./images/desktop_version/frame_headline/Group-35.png" class="desktop-group-35" alt="Group 35 image">
+    </div>
+
+    <!-- Desktop version header -->
+    <header class="header">
+        <ul class="menu">
+            <li class="menu menu-items1"><a href="#"></a>Portfolio</li>
+            <li class="menu menu-items2"><a href="#"></a>About</li>
+            <li class="menu menu-items3"><a href="#"></a>Contact</li>
+        </ul>
+    </header>
+
+    <!-- Desktop version headline -->
+    <section class="desktop-headline">
+        <div class="desktop-headline-first-text">
+            <h2 class="desktop-headline-heading">Hey there. I’m Norbi<br><span>I’m a software developer</span></h2>
+        </div>
+
+        <div class="desktop-headline-supporting-text">
+            <p class="desktop-headline-secondary-text">
+                I can help you build a product , feature or website Look through some of my work and <br> experience! If you like what you see and have a project you need coded, <br>
+                don’t hestiate to contact me.
+            </p>
+        </div>
+
+        <ul class="desktop-social-media-icons">
+            <li><a href="https://github.com/norbiboy1"><img src="./images/NormalButton2/Tertiary/Icons/github.png" class="github" alt="Github icon"></a></li>
+            <li><a href="https://www.linkedin.com/in/siko-norbert/"><img src="./images/NormalButton2/Tertiary/Icons/linkedin.png" class="linkedin" alt="Linkedin icon"></a></li>
+            <li><a href="#"><img src="./images/NormalButton2/Tertiary/Icons/angelis.png" class="angelis" alt="Angelis icon"></a></li>
+            <li><a href="https://twitter.com/NorbertSiko"><img src="./images/NormalButton2/Tertiary/Icons/twitter.png" class="twitter" alt="Twitter icon"></a></li>
+            <li><a href="#"><img src="./images/NormalButton2/Tertiary/Icons/Vector.png" class="vector" alt="Vector icon"></a></li>
+        </ul>
+
+        <div class="desktop-normal-button">
+            <button type="button" class="btn2">
+                <img src="./images/NormalButton2/Secondary/Disabled.png" class="desktop-disabled-button" alt="Arrow down icon">
+            </button>
+        </div>
+    </section>
+
+    <!-- Desktop version work section -->
+    <section class="desktop-work">
+        <div class="desktop-recent-works">
+            <p class="desktop-primary-text">
+                My Recent Works
```

Comment:
- [ ] You have used position absolute here to layout the elements, that is not recommended, please use flexbox or grid
![image](https://user-images.githubusercontent.com/62705947/186936838-a38ab9e9-d573-4ad2-9b8e-e4ac215d59e0.png)